services/prefect/deployment.py

```diff
@@ -0,0 +1,43 @@
+import logging
+import click
+from services.prefect.flows.generated_flows import deploy_scraper_flows
+from services.prefect.flows.github_pipeline_trigger import (
+    deploy_github_pipeline_trigger_flow,
+)
+from services.prefect.flows.update_api_view import deploy_update_parquet_flow
+from services.prefect.flows.clean_sql import deploy_clean_sql_flows
+
+_logger = logging.getLogger(__name__)
+_logger.setLevel(logging.INFO)
+
+
+@click.group()
+def main():
+    pass
+
+
+@main.command()
+@click.option(
+    "--choice",
+    "-c",
+    type=click.Choice(
+        ["all", "scraper", "clean-sql", "update-parquet", "github-pipeline-trigger"]
+    ),
+)
+def deploy_flows(choice: str):
```

Comment:
allows us to deploy flows from the command line with `python -m services.prefect.deployment deploy-flows -c all`